app/assets/stylesheets/pygments.scss

```diff
@@ -63,3 +63,33 @@
 .highlight .ld, /* Literal.Date */
 .highlight .m   /* Literal.Number */
 { color: #ca1655; }
+
+.highlight {
+  position: relative;
+
+  &:focus, &:hover {
+    .highlight__button {
+      visibility: visible;
```

Comment:
I suggest we can use `opacity` instead of `visibility` to toggle the copy button because you can tab to a 0 opacity button (whereas you can't do so with `visibility: hidden;` or `display: none;`.

If we take the `opacity` approach, we can remove the `tabindex="1"` in the `div.highlight` element.

<img width="776" alt="Screen Shot 2023-01-09 at 11 29 21 pm" src="https://user-images.githubusercontent.com/7202667/211308227-3b6fd2fa-1b33-465d-bdb0-1b5f2b579b07.png">

☝️ Screenshot on my local dev, using the `opacity` approach, and I can tab to focus on the copy button without mouseover.